cmd/flags.go

```diff
@@ -0,0 +1,107 @@
+// Package cmd for handling command line arguments
+package cmd
```

Comment:
we need to remove flags and use a file for each command separately
suggestion: cobra